### PR TITLE
Enforce nvm version      GsSoC'23

### PR DESCRIPTION
**Pull request by Nikhil1017**
As per Issue #414

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18
```
Comment (lalitmee) [inline .nvmrc:1]: Are we using `nvm` somehow?

**Comment by lalitmee** [comment]
@Nikhil1017, why all of your PRs almost having the same thing?

**Comment by Nikhil1017** [comment]
@lalitmee yes we are using nvm to set node version globally so that there are conflicts due to version in the future, this PR is for node version and next one is for node package manager version, I am splitting the PRs as the project admin asked me to do so.

**Comment by Nikhil1017** [comment]
@Vaishnavi2701-mk

**Review by lalitmee — COMMENTED** [review]
I think we can remove `package-lock.json` from the changes.

**Comment by Nikhil1017** [comment]
@lalitmee I have removed package-lock.json from the changes

**Review by lalitmee — APPROVED** [review]

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy

**Pull request closed without merging**